scss/_mixins.scss

```diff
@@ -31,11 +31,16 @@
 	// It still needs to use the build service  <-- what does this comment mean?
 	background-image: url($service-url + $query + "&format=png&width=#{$container-width}")\9;
 
-	width: $container-width + 0px;
-	@if ($container-height == null) {
-		$container-height: $container-width;
+	// Added so developers can prevent dimension styles from being output.
+	// This is because in the previous `oFtIconsGetSVG` mixin, the dimension
```

Comment:
Alice asked me to add the comment as otherwise I think the code looks a bit out of place. I'll reword it though.